<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscodeDialog</class>
 <widget class="QDialog" name="TranscodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transcode Music</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/64x64/strawberry.png</normaloff>:/icons/64x64/strawberry.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="layout_transcodedialog">
   <item>
    <widget class="QGroupBox" name="input_group">
     <property name="title">
      <string>Files to transcode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="files">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Directory</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Import Directory</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QPushButton" name="add">
          <property name="text">
           <string>Add...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="import">
          <property name="toolTip">
           <string>Add all tracks from a directory and all its subdirectories</string>
          </property>
          <property name="text">
           <string>Import...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="output_group">
     <property name="title">
      <string>Output options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Audio format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="format">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="options">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="destination">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Alongside the originals</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="select">
        <property name="text">
         <string>Select...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="preserve_dir_structure">
        <property name="text">
         <string>Preserve directory structure in output directory (import only)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progress_group">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="layout_progress">
        <item>
         <widget class="QLabel" name="progress_text">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="details">
          <property name="text">
           <string>Details...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>files</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>import</tabstop>
  <tabstop>format</tabstop>
  <tabstop>options</tabstop>
  <tabstop>destination</tabstop>
  <tabstop>select</tabstop>
  <tabstop>details</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>TranscodeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>TranscodeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
